.travis/matrix-install.sh

```diff
@@ -5,27 +5,23 @@ source .travis/utils.sh
 
 echo "Matrix params: MATRIX_TYPE=${MATRIX_TYPE:?Empty value for MATRIX_TYPE}, BOWER=${BOWER:-no}"
 
-if [ "${MATRIX_TYPE}" = "python" ]; then
-    pip install --exists-action w --timeout 60 --requirement=requirements/test-requirements.txt
+if [ "${MATRIX_TYPE}" = "python" ] || [ "${MATRIX_TYPE}" = "python-sharded" ]; then
 
-    setup_elasticsearch
+    sleep 10  # kafka is slow to start up
     setup_kafka
```

Comment:
(i'm planning on moving topic creation to the app layer which I think should remove the need for this and make tests more robust)